lib/stream.js

```diff
@@ -85,3 +85,27 @@ if (version[0] === 0 && version[1] < 12) {
     };
   }
 }
+
+Object.defineProperty(Stream.prototype, 'readableObjectMode', {
```

Comment:
Isn't it better to move this to `_stream_readable.js`. `readableLength` is already defined there.

Reply:
Yes, these should be defined in `_stream_readable.js` and `_stream_writable.js`.